openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json

```diff
@@ -489,92 +490,92 @@
+    "configure-webhook-message": "OpenMetadata can be configured to automatically send out event notifications to registered webhooks. Enter the Webhook Name, and an Endpoint URL to receive the HTTP callback on. Use Event Filters to only receive notifications based on events of interest, like when an entity is created, updated, or deleted; and for the entities your application is interested in. Add a description to help people understand the purpose of the webhook and to keep  track of the use case. Use advanced configuration to set up a shared secret key to verify the webhook events using HMAC signature.",
+    "configure-webhook-name-message": "OpenMetadata can be configured to automatically send out event notifications to registered {{webhookType}} webhooks through OpenMetadata. Enter the {{webhookType}} webhook name, and an Endpoint URL to receive the HTTP callback on. Use Event Filters to only receive notifications for the required entities. Filter events based on when an entity is created, updated, or deleted. Add a description to note the use case of the webhook. You can use advanced configuration to set up a shared secret key to verify the {{webhookType}} webhook events using HMAC signature.",
     "confirm-delete-message": "Are you sure you want to permanently delete this message?",
     "create-new-glossary-guide": "A Glossary is a controlled vocabulary used to define the concepts and terminology in an organization. Glossaries can be specific to a certain domain (for e.g., Business Glossary, Technical Glossary). In the glossary, the standard terms and concepts can be defined along with the synonyms, and related terms. Control can be established over how and who can add the terms in the glossary.",
-    "create-or-update-email-account-for-bot": "Changing account email will update or create a new bot user",
+    "create-or-update-email-account-for-bot": "Changing the account email will update or create a new bot user.",
     "created-this-task-lowercase": "created this task",
-    "data-asset-has-been-action-type": "Data asset has been {{actionType}}",
+    "data-asset-has-been-action-type": "Data Asset has been {{actionType}}",
     "data-insight-page-views": "Displays the number of times a dataset type was viewed.",
     "data-insight-subtitle": "Get a single pane view of the health of all your data assets over time.",
     "dbt-run-result-http-path-message": "dbt run results http path to extract the test results information.",
```

Comment:
Would it be correct to say

dbt runs the results on an http path to extract the test results.